DALI_EXTRA_VERSION

```diff
@@ -1 +1 @@
-d61722e9fa6df5379cba68941e3f94bff9814def
+db6402db00fc3a96fe060e62fb19091be7891f1a
```

Comment:
The changes in DALI_extra are now merged. You can now change this hash the top of master branch